analyses/oncoprint-landscape/00-map-to-sample_id.R

```diff
@@ -166,8 +166,13 @@ if (!is.null(opt$independent_specimens)) {
 
 message("Preparing MAF file...")
 
-# MAF files already capture sample IDs
+# join the sample_id information to the MAF file and then set as the tumor
+# sample barcode
 maf_df <- maf_df %>%
+  inner_join(select(histologies_df,
```

Comment:
I am not as familiar with the oncoprint stuff, so @cbethell should look at this since she'll have a better idea of what's going on in this context.